subset/network/test_network

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+REPORT=/tmp/report.txt
+
+MONITOR=/scans/monitor.pcap
+
+python network_tests.py connection.dhcp_long $MONITOR $TARGET_IP
+python network_tests.py connection.min_send $MONITOR $TARGET_IP
+
+cat report.txt >> $REPORT
```

Comment:
@ewRedstone It'd be great to return a non-zero from this test if it doesn't pass e.g.

https://github.com/grafnu/daq/blob/master/subset/pentests/test_nmap#L59

This means the dashboard shows a red box if it fails rather than a green box for every result type e.g.

![image](https://user-images.githubusercontent.com/5684825/64166442-07c31e00-ce3f-11e9-800a-52d1ac1167a8.png)

Reply:
Please can you elaborate further. I think there are many different case statements that could be ?

Please can you add clear instructions to the [DAQ test development ](https://docs.google.com/presentation/d/1hCb2OvPU7xmRwibw5oClrXIrwV8fhMobUfZT9HsGFzY/edit#slide=id.g5e7558c38c_0_46)

Reply:
@pisuke I think you had a document defining all the different statuses once ? 
@InBrewJ and I had a discussion resulting to the potential of using a number 0 - 127 to denote the status of different tests encompassed within a module.
0 = good state all tests passed
1 = test 1 fail, test 2 pass
2 = test 1 pass, test 2 fail 
3 = test 1 pass, test 2 pass

Please advise further.

Reply:
@InBrewJ advised that if any test fails then return 1, if all tests pass return 0